resources/home/dnanexus/generate_workbook.py

```
@@ -44,6 +44,18 @@ def main(**kwargs):
 
         inputs[name]["data"] = data
 
+    germline_df = excel.process_reported_variants_germline(
+        inputs["reported_variants"]["data"],
+        inputs["clinvar"]["data"],
+    )
+    somatic_df = excel.process_reported_variants_somatic(
```

Ruff (0.8.2)

Local variable `somatic_df` is assigned to but never used

Remove assignment to unused variable `somatic_df`

(F841)